<html>
	<head>
		<title>BioXSD | BioJSON | BioYAML - Unified exchange formats for biomolecular sequences, features, alignments, references</title>
		<style type="text/css">
			html	{ background-color: whitesmoke }
			body	{ background-color: white; padding-left: 1.1%; padding-right: 1.1%; padding-top: 8px; padding-top: 0.5em; padding-bottom: 8px; padding-bottom: 0.5em; min-width: 480px; min-width: 29em; max-width: 960px; max-width: 58em; margin: auto }
			h1::before	{width: 51%; display: block; overflow: hidden; content: ""}
			hr		{ lightgray; background-color: whitesmoke; clear: both; padding-top: 0.5px; padding-top: 0.02em; padding-bottom: 0.5px; padding-bottom: 0.02em }
			div		{ margin-left: 1.1%; margin-right: 1.3%; line-height: 1.4; text-align: justify }
			p.tabbed	{ margin-left: 20px; margin-left: 1.3em; text-align: left }
		</style>
	</head>
	
	
	
	
	<body>
	
		<img alt="BioXSD | BioJSON | BioYAML" src="bxLogo.png" style="float: left; margin-left: 1.1%; margin-right: 26px; margin-right: 1.64em; margin-top: 12px; margin-top: 0.75em; margin-bottom: 48px; margin-bottom: 3em; width: auto; height: auto; max-width: 60%; max-height: 131.2px; max-height: 8em; min-width: 272px; min-width: 16em"/>
		
		<!--<p style="border: 1px solid green; width: 51%; display: block; overflow: hidden; margin: 0"/>doesn't work in firefox, needs ::before pseudo-element instead (or alternatively some further positioning hacking of this p)-->
		
		<h1 style="margin-left: 1.1%; margin-right: 1.3%; margin-top: 3px; margin-top: 0.18em; margin-bottom: 22px; margin-bottom: 1.4em; line-height: 1.3; font-size: 1.72em">A unified data model for biomolecular sequences, features, alignments, references</h1>
		
		<h2 style="color: #424242; margin-left: 1.1%; margin-right: 1.3%; margin-top: 16px; margin-top: 1em; margin-bottom: 16px; margin-bottom: 1em; line-height: 1.3; font-size: 1.34em">Suitable as <!--an--> exchange formats for Web services, web applications, and programmatic libraries</h2>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/>
			<a href="#News">News</a><br/>
			<a href="#About">About</a><br/>
			<a href="#XSD">The XML Schema</a><br/>
			<a href="#Documentation">Documentation</a><br/>
			<a href="#ExampleData">Example data</a><br/>
			<a href="#ExampleWorkflows">Example workflows</a><br/>
			<a href="#Tools">Compliant tools</a><br/>
			<a href="#Ongoing">Ongoing development</a><br/>
			<a href="#Credits">Credits</a><br/>
			<a href="#Contact">Contact &amp; follow</a><br/>
			<br/>
		</div>		

		<hr/><!--================================================================================================================================-->

		<div>
			<br/><br/>
			<a name="News"/><h2>News</h2>
			<br/><br/>
		
			<p>
				<big><u><b>GTrack became open</b></u></big> &nbsp&nbsp <em>6 Mar 2018</em><br/>
				<a href="http://gtrack.no"><b>GTrack</b></a> is from now on available under the <a href="http://creativecommons.org/licenses/by-sa/4.0/"><b>CC BY-SA 4.0</b></a> licence, and developed in its new <a href="https://github.com/gsuite/gtrack">GitHub repo</a>.
			</p>
		
			<p>
				<big><u><b>BioXSD | GTrack - A service developed by ELIXIR Norway</b></u></big><br/>
				It has been confirmed that ELIXIR Norway will further develop and deliver BioXSD and <a href="http://gtrack.no">GTrack</a> as a family of interoperable data formats and supportive tools.
				<br/>The family of interoperable generic data formats for bioinformatics and genomics comprises BioXSD, BioJSON, BioYAML, the tabular GTrack, binary tabular BTrack, and the integrative GSuite, with supportive conversion tools under development.
				<br/>The aim is to provide a selection of generic, mutually loss-lessly convertible formats suitable for various usage scenarios, such as providing tools, APIs, web apps, or databases, or integrating bioinformatics|genomics data.
			</p>
			
			<br/>
		
			<p>
				<big><u><b>BioXSD | BioJSON | BioYAML - Towards a unified data model with multiple serialisation options</b></u></big><br/>
				BioXSD is a construction site again. Current developments focus on employing XML Schema version 1.1 and JSON Schema, and enabling BioXSD to function as a single data model with multiple choices of tree-based exchange formats (serialisations).
				<br/>Highest on the priority list is adding JSON and YAML as serialisation alternatives to XML, and implementing smooth interoperability between BioXSD, BioJSON, BioYAML, and <a href="http://gtrack.no">GTrack</a> (the unified tabular format for sequence features).
			</p>
			
			<br/>
		
			<p>
				<big><u><b>BioXSD version 1.1, release 1.1.3</b></u></big><br/>
				The next stable minor release of BioXSD 1.1, version BioXSD 1.1.3, was released on the 15th of May, 2015.
				<br/>As all minor releases of BioXSD are compatible with all previous and future minor releases within the last major version/namespace, the <em>BioXSD 1.1.3</em> is "backwards and forwards compatible" with all <b>BioXSD 1.1</b> data instances <small>(<i>i.e.</i> less restricted than the previous versions, and extended with non-mandatory new attributes/elements)</small>.
				<br/>For the list of changes, please refer to the CHANGELOG in the <a href="http://bioxsd.org/BioXSD-1.1.xsd"><em>BioXSD-1.1.xsd</em></a> file.
				<br/>Developments towards BioXSD version 1.2 (the next major, "backwards <em>in</em>compatible" release) are ongoing in parallel, currently in a pre-implementation stage experimenting with essential design decisions. The 1.2 developments at the moment focus on interoperability with JSON, GTrack, JSON schema, BSON, EXI, RDF (and potentially later also other Semantic Web formats, AVRO/Hadoop, RELAX NG or other alternative schema languages), with the PROV provenance metadata, and migration to XML Schema version 1.1.
			</p>
			
			<br/>
		
			<p>
				<big><u><b>BioXSD becoming more "open"</b></u></big><br/>
				BioXSD 1.1 is now available under a <b>CC BY-SA 4.0</b> licence! Please note also the new Code of Conduct, outlined in <a href="http://bioxsd.org/BioXSD-1.1.xsd"><em>BioXSD-1.1.xsd</em></a>. In addition, we finally started the long-desired transition to the new source-files repository at <a href="http://github.com/bioxsd/bioxsd"><b>http://github.com/bioxsd/bioxsd</b></a>. We believe that this will bring more transparency to the developments and maintenance of BioXSD, and decrease the friction for participating in the future evolution of BioXSD and related software.
			</p>
			
			<br/>
		
			<p>
				<big><u><b>Enhanced and standardised provenance metadata are under development</b></u></big><br/>
				We are improving the provenance metadata in BioXSD, by implementing a rich but simple model that will be aligned with the W3C <a href="http://www.w3.org/2011/prov/wiki/Main_Page">PROV</a> standard. It will most likely be part of the next minor release.
				<!--<br/>-->Thanks shall among others be sent to the <a href="https://bcbio.wordpress.com/2013/07/18/summary-from-bioinformatics-open-science-codefest-2013-tools-infrastructure-standards-and-visualization/">OpenBio Codefest 2013</a> for input and discussions.
				<!--<br/>A new minor release of <b>BioXSD 1.1</b>, fully backwards compatible with all valid BioXSD 1.1 data, will include the provenance enhancements and standardisation, and a couple of more improvements. It will be the version <em>1.1.3</em>.-->
				<br/>For <!--urgently--> desired additions into the next release, please contact <a href="mailto:developers@bioxsd.org">developers@bioxsd.org</a><!-- as soon as possible-->.
			</p>
			
			<br/>
		
			<p>
				<big><u><b>BioXSD version 1.1, release 1.1.2</b></u></big><br/>
				A minor release of BioXSD 1.1, the version BioXSD 1.1.2, was released on the 13th of May, 2013.
				<br/>As all minor releases of BioXSD are compatible with all previous minor releases within the last major release, the <em>BioXSD 1.1.2</em> is backwards compatible with <b>BioXSD 1.1</b> data <small>(<em>i.e.</em> less restricted)</small>.
				<br/>For the list of changes, please refer to the CHANGELOG in the <a href="http://bioxsd.org/BioXSD-1.1.xsd"><em>BioXSD-1.1.xsd</em></a> file.
			</p>
			
			<br/>
		
			<p>
				<big><u><b>The 50 BioXSD-compatible Web services challenge</b></u></big><br/>
				Volunteers are still warmly welcome to join the <em>50 BioXSD-compatible Web services</em> challenge!
				<br/>The aim is to provide compatible Web services and other tools that adopt BioXSD as one of their input/output formats.
				<br/>Implementations have already been ongoing at CBS in Greater Copenhagen (Denmark), Rostlab in Greater Munich (Germany), IBCP in Lyon (France), CBU in Bergen (Norway), and at a couple of more sites.
				<br/><!--<i><small>-->For more details, help, and consultation, please contact <a href="mailto:support@bioxsd.org">support@bioxsd.org</a>.<!--</small></i>-->
			</p>
			
			<br/>
		</div>
			
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="About"/><h2>About</h2>
			<br/><br/>
			
			<p><!--<big>-->BioXSD is a data model defining exchange formats for <em>basic</em> bioinformatics types of data: <b>sequences</b>, <b>alignments</b>, <b>feature records</b> with associated data and metadata.
			BioXSD aims to serve as a common, canonical tree-structured format (XML, JSON, YAML, binary) for the basic bioinformatics data.
			Specialised standard XML-schema formats such as for example SBML, MAGE-ML, GCDML, PDBML, PSI MI MIF, PhyloXML or NeXML are orthogonal efforts and should be used wherever applicable. BioXSD aims at filling the gap between these specialised XML formats.</p><!--Add links!-->
			
			<p>A canonical data format does not mean "the only format", but an exchange format that can be common to several tools (as one of multiple formats the tools are supporting). Tools can produce and consume BioXSD directly, or BioXSD can be used as an intermediate canonical format, rich enough to enable conversions among diverse formats. Using common exchange format enables smooth integration of compatible tools into analysis workflows.</p>
			
			<p>BioXSD is a rich but not too complicated data model and exchange format, aiming to enable deployment of globally and smoothly interoperable bioinformatics tools on the World Wide Web of Services.<!--<br/>-->
			BioXSD supports HTTP ("REST") Web services and WS-I compliant SOAP Web services, and interoperates with ordinary HTTP (REST), SOAP, and XML libraries for common programming languages. No other infrastructure than standard HTTP and XML is necessary for using BioXSD-compatible Web services. In the near future, we would like to add <em>smooth</em> support for JSON, YAML, and eventually other data-serialisation languages, as alternatives to using BioXSD in XML.</p>
			
			<p><!--<em>-->BioXSD is a "blue-collar" initiative originating from the inside of the bioinformatics community, initiated among others by the <a href="http://www.embracegrid.info">EMBRACE</a> project partners and the participants of the <a href="http://hackathon.dbcls.jp">1<sup>st</sup> DBCLS BioHackathon</a>, in 2008.<!--</em>--></p>
			
			<br/>
			<p><img alt="Interoperability standards according to EMBRACE" src="embraceInteroperability.png" style="display: block; margin-left: auto; margin-right: auto; width: auto; height: auto; max-width: 100%; max-height: 300px; max-height: 18em; min-width: 480px; min-width: 29em"/></p>
			<p/>
			<p style="color: #424242; text-align: center; margin-top: 24px; margin-top: 1.5em"><u>The portfolio of standards according to EMBRACE</u></p>
			<br/><br/>
			
			<p><!--<big>-->BioXSD data-type definitions are annotated with <a href="http://edamontology.org">EDAM</a> and with a few main Semantic-Web vocabularies. BioXSD thus offers ready-made building blocks for tool interfaces with a globally defined, controlled meaning (semantics).<!--</big>--></p>
	
			<br/><br/>
			
			<p>BioXSD has been developed by analysing existing requirements, tools, Web services, data formats, and ontologies.<!--<br/>-->
			Feasibility was tested at different pilot providers, using diverse libraries and programming languages.</p>
			
			<p><!--<small>-->BioXSD types can be used directly if applicable; or can be included in other standard or custom types, extended or restricted.
			With tools that use other or proprietary formats, BioXSD can be used as a canonical intermediate exchange format.<!--</small>--></p>
	
			<br/><br/><!--<br/>-->
			<p><b><big>Open collaboration within the community:</big></b> BioXSD welcomes feature requests, suggestions, and new collaborations!</p>
			<p>
				To submit your proposals, suggestions and requirements, please post an issue at <a href="http://github.com/bioxsd/bioxsd">http://github.com/bioxsd/bioxsd</a>, or post to a new or existing topic at <a href="http://groups.google.com/group/bioxsd">http://groups.google.com/group/bioxsd</a><!-- (<a href="http://groups.google.com/group/bioxsd/subscribe">please join</a> first)-->, or write to <a href="mailto:developers@bioxsd.org">developers@bioxsd.org</a>. Please see <a href="#Contact">contact details</a>.
			</p>
			<br/><br/><br/>
	
			<p><big><b><a href="bxPoster.pdf">A BioXSD poster</a></b></big></p>
			
			<br/><br/><br/>
				
				<p><big><b>Publications:</b></big></p>
				<br/>
				<p><i>If using BioXSD in work published or advertised as a scholarly publication, please reference:</i></p>
				<p>
					Kala&#353;, M., Puntervoll, P., Joseph, A.,
					Barta&#353;evi&#269;i&#363;t&#279; (now Karosiene), E.,
					T&#246;pfer, A., Venkataraman, P., Pettifer, S., Bryne, J.C.,
					Ison, J., Blanchet, C., Rapacki, K. and Jonassen, I.
					(2010).
					<a href="http://bioinformatics.oxfordjournals.org/content/26/18/i540.full">BioXSD: the common data-exchange format for everyday bioinformatics web services.</a>
					<i>Bioinformatics</i>, <b>26</b>, i540-i546.
					<br/><small>doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btq391">10.1093/bioinformatics/btq391</a>&#160;&#160; PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/20823319">20823319</a></small>
				</p>
				<br/>
				<p><i>If using the optimised sequence/genome feature representation (since BioXSD 1.1), please reference also:</i></p>
				<p>
					Gundersen, S., Kala&#353;, M., Abul, O.,
					Frigessi, A., Hovig, E. and Sandve, G.K.
					(2011).
					<a href="http://www.biomedcentral.com/1471-2105/12/494">Identifying elemental genomic track types and representing them uniformly.</a>
					<i>BMC Bioinformatics</i>, <b>212</b>, 494.
					<br/><small>doi: <a href="http://dx.doi.org/10.1186/1471-2105-12-494">10.1186/1471-2105-12-494</a>&#160;&#160; PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22208806">22208806</a></small>
				</p>
				<br/>
				<p>BioXSD 1.1 has been developed hand-in-hand with <a href="http://gtrack.no"><b>GTrack</b></a>, the unifying tabular format for sequence/genome features. Both formats use the same tactics for optimising size of diverse kinds of feature records, via flexible options for recording features' positions and other data.</p>
				
			<br/><br/>
		</div>	
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="XSD"/><h2>The XML Schema</h2>
			<br/><br/>
			
			<p><big><b>BioXSD 1.1</b></big> (namespace <small><code>http://bioxsd.org/BioXSD-1.1</code></small>) is available at <a href="http://bioxsd.org/BioXSD-1.1.xsd"><em>http://bioxsd.org/BioXSD-1.1.xsd</em></a>. This latest stable version is available for implementations, actively maintained, and open to further developments. New requirements and suggestions for changes are welcome and may be reflected in future versions. <!--</big>--></p>
			
			<p><a href="http://bioxsd.org/BioXSD-1.1.xsd"><em>http://bioxsd.org/BioXSD-1.1.xsd</em></a> is the canonical schema location to be</p>
			<ul>
				<li>used for XML data instances
					<p class="tabbed"><small><code>xsi:schemaLocation="http://bioxsd.org/BioXSD-1.1 http://bioxsd.org/BioXSD-1.1.xsd"</code><br/><i>working <a href="sequenceRecord.xml">example</a></i></small></p>
				</li>
				<li>imported in document XSDs (including Web services' WSDLs)
					<p class="tabbed"><small><code>&lt;xs:import namespace="http://bioxsd.org/BioXSD-1.1" schemaLocation="http://bioxsd.org/BioXSD-1.1.xsd"/></code><br/><i>working example <a href="classifierOutput.xsd">XSD</a> and <a href="classifierOutputExample1.xml">instance</a></i></small></p>
				</li>
				<li>or to be synchronised with its future releases in case of publishing custom-annotated or custom-modified BioXSDs.</li>
			</ul>
			
			<p><small>
			(Note: There is no "worked-around" version of BioXSD 1.1 available for SOAP Web service providers using Python ZSI for their backend SOAP stack. It can be made available if required, please contact <a href="mailto:support@bioxsd.org">support@bioxsd.org</a>.)
			</small></p>
			
			<br/><br/>
			
			<p><!--<big>-->Older <b>BioXSD 1.0</b> (namespace <small><code>http://bioxsd.org/BioXSD-1.0</code></small>) is available at <a href="http://bioxsd.org/BioXSD-1.0.xsd"><em>http://bioxsd.org/BioXSD-1.0.xsd</em></a>. <!--</big>--><!--This is the canonical Schema location to be imported in document XSDs (such as in Web services' WSDLs) or to be synchronised with.--></p>
			
			<p><small>
			(For SOAP Web service providers using Python ZSI for their backend SOAP stack: Due to some important basic features missing in the Python Zolera Soap Infrastructure (ZSI) library, a special version for generating ZSI code is at
			<a href="http://bioxsd.org/BioXSD-1.0.zsi.Workaround.xsd"><em>http://bioxsd.org/BioXSD-1.0.zsi.Workaround.xsd</em></a>.
			Do not forget to get the <a href="http://bioxsd.org/ZSI-2.1_a2-py2.5.egg">ZSI patch</a> for the <a href="http://cagrid.org/display/knowledgebase/Develop+Python+Clients">empty-complexType bug</a>.
			This xsd is "SOAP-compatible" with the normal xsd. It means that the services in Python should be generated from WSDLs importing the <em>BioXSD-x.x.zsi.Workaround.xsd</em> Schema,
			but WSDLs of the deployed services should then be importing the normal <em>BioXSD-x.x.xsd</em> Schema.)
			</small></p>
			
			<br/><br/>
			
			<p>The source files and releases of the BioXSD schema, of supportive utility software, and of documentation are in transition to <a href="http://github.com/bioxsd/bioxsd">http://github.com/bioxsd/bioxsd</a>, and will gradually appear there.</p>
			
			<br/>
			
			<p>
				<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">
					<img alt="Creative Commons Attribution-ShareAlike 4.0 International License" style="border-width: 0; width: 100px; width: 72pt" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png"/>
				</a>
				<br/>
				<a href="http://bioxsd.org/BioXSD-1.1.xsd"><!--<a xmlns:cc="http://creativecommons.org/ns#" xmlns:dct="http://purl.org/dc/terms/" href="http://bioxsd.org" rel="cc:attributionURL" property="dct:title">-->BioXSD</a> data models and schemata are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a> (CC BY-SA 4.0). Please note also the <a href="http://bioxsd.org/BioXSD-1.1.xsd">Code of Conduct</a> for derived work, outlined at the top of the <a href="http://bioxsd.org/BioXSD-1.1.xsd">http://bioxsd.org/BioXSD-1.1.xsd</a> file.
			</p>
			
			<!--<p>BioXSD Schemas are available under the <b>Creative Commons BY-ND 3.0</b> license with additionally allowed inclusion, extensions and restrictions in user's XML namespace. Contributions to new canonical versions, in the <i>bioxsd.org</i> XML namespace, are welcome under supervision of the BioXSD consortium (in order to keep BioXSD a common, canonical data model).</p>
			
			<br/><br/>-->
			
			<p>For release information including CHANGELOG, please refer to the bottom of the XSD files.</p>
	
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="Documentation"/><h2>Documentation</h2>
			<br/><br/>
	
			
			<p>A concise <big><b>quick overview guide</b></big> to the BioXSD format is at <a href="QuickReference"><!--<big>--><em>./QuickReference</em><!--</big>--></a>.</p>
			
			
			<br/><br/>
	
			
			<p>Full reference documentation of BioXSD version 1.1 is at <a href="technicalDocumentation/BioXSD-1.1"><!--<big>--><em>./technicalDocumentation/BioXSD-1.1</em><!--</big>--></a>.</p>
			
			
			<br/><br/>
			
			<p>Documentation of BioXSD version 1.0 is available at <a href="technicalDocumentation/BioXSD-1.0"><!--<big>--><em>./technicalDocumentation/BioXSD-1.0</em><!--</big>--></a>.</p>
	
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="ExampleData"/><h2>Examples of data</h2>
			<br/><br/>
			
			<p>A <a href="accession.xml">UniProt accession as a BioXSD 1.1 element</a> - a toy example of an "inline" instance of a simple type.</p>
			
			<p>A BioXSD 1.1 <a href="sequenceRecord.xml+json+yaml.xml">sequence record</a> is an example "inline" instance of a complex type: in <a href="sequenceRecord.xml">XML</a>, <a href="sequenceRecord.json">BioJSON</a>, and <a href="sequenceRecord.yaml">BioYAML</a>.</p>
			
			<p><a href="exampleFeatureRecords.html">Examples of feature data</a> represented in <!--<a href="http://bioxsd.org/BioXSD-1.1beta1.xsd">--><!--<b>-->BioXSD 1.1<!--</b>--><!--</a>--> format.</p>
			
			<br/><br/>
			
			<p><a href="bxExamples(1.0).xml">Examples of diverse types of bioinformatics data</a> represented in the older <!--<a href="http://bioxsd.org/BioXSD-1.0.xsd">--><!--<b>-->BioXSD 1.0<!--</b>--><!--</a>--> format are available in the referred example file. The example file contains numerous examples of sequence records, annotated sequences, and multiple-sequence alignments.</p>
			
			<!--<p><small>Examples of data represented in beta <a href="http://bioxsd.org/BioXSD-0.4.xsd">BioXSD 0.4</a> format are <a href="bxExample(0.4).xml">here</a>.
			(And examples in --><!--<a href="http://bioxsd.org/BioXSD-0.3u.xsd">--><!--BioXSD 0.3--><!--</a>--><!-- <a href="bxExample(0.3).xml">here</a>)</small></p>-->
			
			
			
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="ExampleWorkflows"/><h2>Example workflows</h2>
			<br/><br/>
			
			<p><a href="exampleWorkflows">Example workflows</a> (analysis pipelines) combine multiple bioinformatics Web services using BioXSD.</p>
			
			<p>Workflows show that such services are <i>smoothly compatible</i>.</p>
			
			
			
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="Tools"/><h2>Compliant tools</h2>
			<br/><br/>
	
			<p>
				Providers of bioinformatics tools who started adopting BioXSD as pilot users:
			</p><br/>
			<p><big><b>Web services:</b></big></p>
			<ul>
				<li><em>CBS, DTU, Greater Copenhagen, Denmark:</em>
					<a href="http://www.cbs.dtu.dk/ws/MaxAlign/MaxAlign_1_1_ws1.wsdl">MaxAlign</a>,
					<a href="http://www.cbs.dtu.dk/ws/ProP/ProP_1_0_ws0.wsdl">ProP</a>,
					<a href="http://www.cbs.dtu.dk/ws/NetNES/NetNES_1_1_ws0.wsdl">NetNES</a>
					<!--<em>(more under development)</em>-->
				</li>
				<li><em>CBU, U Bergen, Norway:</em>
					<a href="http://www.bccs.uni.no/services/blastws/BlastPublic_2_2_21_ws0.wsdl">BLAST</a>
					<!--<em>(under development:</em> MAFFT, Mblosum<em>)</em>-->
				</li>
				<li><em>IBCP, CNRS, Lyon, France:</em>
					<a href="http://gbio-pbil.ibcp.fr/Tools/ClustalW_bx/wsdl">ClustalW</a>,
					<a href="http://gbio-pbil.ibcp.fr/Tools/Blast_bx/wsdl">BLAST</a>,<!--Does work?-->
					<a href="http://gbio-pbil.ibcp.fr/Tools/GorIV_bx/wsdl">GorIV</a>
					<!--<em>(more under development)</em>-->
				</li>
				<li><em>UBio, CNIO, Madrid, Spain:</em>
					<a href="http://ws.bioinfo.cnio.es/iHOP/EMBRACE/iHOP-SOAP-document-literal-EDAM.wsdl">iHOP</a>
				</li>
			</ul>
			<p><big><b>Software:</b></big></p>
			<ul>
				<li><em>Rostlab, TU Munich, Germany:</em>
					<a href="ftp://rostlab.org/free">FreeContact</a> (in <a href="http://blends.debian.org/med/tasks/bio#freecontact">Debian Med</a>)
				</li>
			</ul>
			<br/>
			<p><small><i>If you started using BioXSD for your software, services, libraries, or other tools, please let us know by sending an email to <a href="mailto:support@bioxsd.org">support@bioxsd.org</a>. For maintenance and support purposes we would love to know about the providers using BioXSD. A registration system may become available in the future.</i></small></p>
			
			<br/><br/>
			
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="Ongoing"/><h2>Ongoing and proposed developments</h2>
			<br/><br/>
			
			<ul>
			<li>update BioXSD to XML Schema version 1.1</li>
			<li>serialisation of BioXSD "data model" in multiple formats (XML, JSON, YAML, GTrack, binary, ... ), with associated validation and conversions</li>
			<li>including JSON Schema, and considerably also other schema languages if desired: Relax NG, Avro, OWL, ...</li>
			<li>tools to translate between BioXSD and other formats (specific textual, tabular, &amp; binary formats, RDF, HDF, ... ), and other supportive tools</li>
			<li>interoperability with the <a href="http://www.w3.org/standards/techs/provenance">PROV</a> standard for provenance metadata</li>
			<li>populating accession types and database names from <a href="http://github.org/edamontology/edamontology">EDAM</a> and <a href="http://drcat.sourceforge.net">DRCAT</a>, and considerably ontology names from the <a href="http://bioportal.bioontology.org">NCBO BioPortal</a> or <a href="http://www.ebi.ac.uk/ontology-lookup">OLS</a></li>
			<li>additional data examples and test-case workflows, including multiple XSD|XML|EXI|JSON|BSON|YAML|HTTP libraries and programming|scripting|workflow languages</li>
			<li>improved BioXSD 101 and an online BioXSD tutorial</li>
			<li>enhancements of the existing BioXSD format. Objective is to support more conveniently and efficiently: measured data (<i>e.g.</i> gene expression), sequence variation (in individuals or population), whole-genome alignments, sequence patterns|profiles|motifs, and some other types of data</li>
			</ul>
			
			<br/><p>Volunteers are also always welcome to contribute by joining the <b><em>50 BioXSD|BioJSON|BioYAML-compatible tools or services</em></b> challenge.</p>
			
			<br/><p>Contributions in all forms are warmly welcome and needed! <small><br/><a href="http://github.com/bioxsd/bioxsd">http://github.com/bioxsd/bioxsd</a>, <a href="https://groups.google.com/group/bioxsd">http://groups.google.com/group/bioxsd</a>, <a href="mailto:developers@bioxsd.org">developers@bioxsd.org</a>, see <a href="#Contact">contact details</a></small></p>
			
			<br/><br/>
			<p><img alt="Ongoing and proposed interoperability developments" src="multiFormatDataModel.png" style="display: block; margin-left: auto; margin-right: auto; width: auto; height: auto; max-width: 100%; max-height: 530px; max-height: 32em; min-width: 480px; min-width: 29em"/></p>
			<p/>
			<p style="color: #424242; text-align: center; margin-top: 24px; margin-top: 1.5em"><u>The ongoing and proposed interoperability developments around BioXSD</u></p>			
			
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<br/><br/>
			<a name="Credits"/><h2>Credits</h2>
			<br/><br/>

			<p>BioXSD was developed as part of multiple collaborative projects. There has never been any funding directed exclusively to BioXSD. The current developments of BioXSD|BioJSON|BioYAML are done together with the developemnts of <a href="http://gtrack.no">GTrack|BTrack|GSuite</a>, as a <b>BioXSD|GTrack</b> service provided by <b>ELIXIR Norway</b>.</p>
			<br/>
			
			<p><big><b>Contribution and advise to the development of BioXSD:</b></big><!--</p>			
			<p>--><br/><em>CBU, II, University of Bergen, Norway:</em>
				<big>Mat&uacute;&scaron; Kal&#13;&#10;a&scaron; Inge Jonassen, </big>P&aring;l Puntervoll<small> (until 2010; now Uni Milj&oslash;, Bergen)</small><!--<big>-->, Jan Christian Bryne<!--</big>--><small> (until 2010; later Oslo University Hospital)</small><!--<big>-->, Armin T&#246;pfer<!--</big>--><small> (until 2011; also CeBiTec, Bielefeld, Germany; later D-BSSE, ETH Z&#252;rich, Basel, Switzerland)</small><!--<big>-->, Prabu Venkataraman<!--</big>--><small> (until 2011; later Fiskeridirektoratet, Bergen)</small>
			<br/><em>CBS, DTU, Greater Copenhagen, Denmark:</em>
				<big>Kristoffer Rapacki, Jon Ison</big> <small>(at CBS, DTU since 2014)</small>, Edita Karosiene <small>(until 2010)</small>
			<br/><em>II, University of Oslo, and Oslo University Hospital, Norway:</em>
				<big>Sveinung Gundersen</big>
			<br/><em>IBCP, CNRS, Lyon, France:</em>
				<big>Christophe Blanchet</big> <small>(now also IFB<!--L'Institut Fran&ccedil;ais de Bioinformatique-->, Gif-sur-Yvette, France)</small>, Alexandre Joseph <small>(until 2010)</small>
			<br/><em>EBI, EMBL, Hinxton, U.K.:</em>
				<big>Jon Ison</big> <small>(until 2013; now see at CBS, DTU above)</small><big>, Rodrigo Lopez</big>
			<br/><em>CS, University of Manchester, U.K.:</em>
				<big>Steve Pettifer</big>
			<br/><em>Rostlab, TUM, Greater Munich, Germany:</em>
				<big>L&aacute;szl&oacute; Kaj&aacute;n</big> <small>(until 2013; now itelligence, Pozna&#324;, Poland)</small>
			<br/><em>... and multiple supporters at diverse research institutions</em>
			</p><!--todo: improve abbrs and maintain links to institutes & people-->
			
			<br/>
			
			<p><big><b>Funding:</b></big>
			<br/>Research Council of Norway (to eSysbio, to the FUGE Bioinformatics Platform, and <b>ELIXIR.NO</b> to the Norwegian Bioinformatics Platform)
			<br/>Villum Foundation (to the Center for Disease Systems Biology)
			<br/>l'Agence Nationale de la Recherche (to HIPCAL)
			<br/>Alexander von Humboldt Foundation (through the German Ministry for Research and Education)
			<br/>European Commission FP6 and FP7 (to EMBRACE and ELIXIR)</p>
			
			<br/><br/>
		</div>
		
		<hr/><!--================================================================================================================================-->
		
		<div>
			<a name="Contact"/><p><b>Contact:</b><!-- <a href="mailto:support@bioxsd.org">support@bioxsd.org</a>--></p>
			
			<p class="tabbed"><a href="mailto:support@bioxsd.org">support@bioxsd.org</a></p>
			
			<p class="tabbed"><a href="mailto:developers@bioxsd.org">developers@bioxsd.org</a></p>
			
			
			<p><b>Connect with us:</b></p>
			
			<p class="tabbed"><a href="https://twitter.com/BioXSD">@BioXSD</a> (<a href="https://twitter.com/intent/follow?original_referer=http%3A%2F%2Fbioxsd.org&amp;region=follow_link&amp;screen_name=bioxsd&amp;tw_p=followbutton">follow</a>)<!--. Alternatively <a href="https://twitter.com/search?q=%23bioxsd">#bioxsd</a>--></p>
			
			<p class="tabbed"><a href="https://github.com/bioxsd/bioxsd">http://github.com/bioxsd/bioxsd</a></p>
			
			<p class="tabbed"><a href="https://groups.google.com/group/bioxsd">http://groups.google.com/group/bioxsd</a>. <a href="https://groups.google.com/group/bioxsd/subscribe">Please join first</a>. For membership without a Google account, just log out of Google before clicking. In such case, please feel free to contact <a href="mailto:bioxsd+manager@googlegroups.com">bioxsd+manager@googlegroups.com</a> after joining, <i>e.g.</i> to change your delivery settings between <i>all_email</i><small>(default)</small>|<i>abridged</i>|<i>digest</i>|<i>no_email</i>. Without a Google account, you can post via email to <a href="mailto:bioxsd@googlegroups.com">bioxsd@googlegroups.com</a> and read at <a href="http://groups.google.com/group/bioxsd">http://groups.google.com/group/bioxsd</a><!--.--></p>
			
			
			<p><small>Please feel free to post and communicate in <b>any</b> language. We will do our best to understand.</small></p>
			
			<p><small><b>Last update: </b>2018-Mar-06</small></p>
			
		</div>
		
	</body>
</html>
